src/core/lib/address_utils/sockaddr_utils.cc

```diff
@@ -60,9 +62,9 @@ static std::string grpc_sockaddr_to_uri_unix_if_possible(
   return absl::StrCat("unix:", unix_addr->sun_path);
 }
 #else
-static std::string grpc_sockaddr_to_uri_unix_if_possible(
+static absl::StatusOr<std::string> grpc_sockaddr_to_uri_unix_if_possible(
     const grpc_resolved_address* /* addr */) {
-  return "";
+  absl::StrCat("Unix socket is not supported.");
```

Comment:
This should return a non-OK status instead of a string.

Reply:
Done.